waiter/src/waiter/scheduler.clj

```diff
@@ -727,7 +731,9 @@
         (doseq [start-time matched-start-times]
           (let [duration (metrics/duration-between start-time current-time)]
             (metrics/report-duration waiter-schedule-timer duration)
-            (metrics/report-duration service-schedule-timer duration)))
+            (metrics/report-duration service-schedule-timer duration)
+            (statsd/histo! metric-group "schedule_time_seconds"
+                           (t/in-seconds duration))))
```

Comment:
Should be consistent with the other timer histograms?
- Our other timers are in millis.  
- For what it's worth, our other timers don't include the time unit in the name.

Reply:
Should we add `startup-time` as well?

Reply:
Should we only populate the statsd metrics from the leader?

Reply:
Whoops, just noticed the WIP tag.

Reply:
Thanks for the feedback.
* Renamed `schedule_time_seconds` → `schedule_time`
* Switched to milliseconds
* Added `startup_time` histo reporting
* Only reporting from leader